services/collaboration/pkg/connector/fileinfo/onlyoffice.go

```diff
@@ -216,6 +227,11 @@ func (oinfo *OnlyOffice) SetProperties(props map[string]interface{}) {
 			oinfo.HidePrintOption = value.(bool)
 		}
 	}
+	exceptedExt := map[string]struct{}{"docx": {}, "xlsx": {}, "pptx": {}, "pdf": {}}
+	ext := strings.ToLower(strings.TrimPrefix(path.Ext(oinfo.BaseFileName), "."))
+	if _, ok := exceptedExt[ext]; !ok {
+		oinfo.Size = nil
+	}
```

Comment:
Couple of things here: the method is expected to just assign the provided values, so doing other things doesn't look good. It might not be the right place to do it.
In addition, I don't think we should be removing the size for some documents when we already have the correct value. If we have a ".doc" file with 6KB, why don't send it? It wasn't sent before because the property wasn't documented (so either not needed or not used on OnlyOffice side), but other than that, I don't think we should be making exceptions.